workshop-prerequisites.html

```diff
@@ -0,0 +1,251 @@
+          <li><a href="index.html#schedule">Schedule</a></li>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">Events<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="https://in.pycon.org/cfp/devsprint-2018/proposals/">Devsprints</a></li>
+              <li><a href="open-spaces.html">Open Spaces</a></li>
+            </ul>
+          </li>
+          <li><a href="index.html#sponsors">Sponsors</a></li>
+          <li><a href="https://in.pycon.org/blog/">Blog</a></li>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">Attend<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="index.html#tickets">Tickets</a></li>
+              <li><a href="index.html#venue">Venue</a></li>
+              <li><a href="travel.html">Traveling to Hyderabad</a></li>
+              <li><a href="hotels.html">Hotels</a></li>
+              <li><a href="https://goo.gl/forms/iqZ8I3xfzcXXLxQq2">Childcare</a></li>
+            </ul>
+          </li>
+          <li><a href="faq.html">FAQ</a></li>
+          <li class="dropdown active">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">More<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="volunteer.html">Volunteer</a></li>
+              <li><a href="https://goo.gl/forms/R4g5sxjZIXezvp7z1">Community Spaces</a></li>
+              <li><a href="coc.html">COC</a></li>
+              <li><a href="index.html#contact">Contact</a></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <!-- Section: about -->
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 class="small-caps center">Workshop Prerequisites</h2>
+        <p class="mb-4">
+          Below are the prerequisites detailed for the workshops either by a Keynote or a Sponsor. All other workshop details can be seen from the <a href="https://in.pycon.org/cfp/2018/proposals/"><strong>CFP</strong></a> itself.
+        </p>
+        <hr>
+        <h3>Jupyter hub and Binder by Carol Wiling</h3>
+        <p class="mb-4">
+            <strong>Abstract -</strong>
+            We'll take an in-depth look at JupyterLab and its use for data science and scientific computing. We'll look at the new user interface functionality available to users of JupyterLab. Later, we will move beyond an individual's workflow with JupyterLab and see how JupyterHub brings JupyterLab to groups of users. We'll wrap up with a discussion of Binder and its ability to provide ephemeral computing environments for Jupyter notebooks and beyond.
+        </p>
+        <hr>
+        <h3>Programming a Quantum Computer using Cirq by Vamsi Krishna Devabathini (Google)</h3>
```

Comment:
Can we put a heading before this to mention these are Sponsored tracks ?

Reply:
ACK